backend/src/bundles/users/user.controller.ts

```diff
@@ -60,6 +69,27 @@ class UserController extends BaseController {
             payload: await this.userService.findAll(),
         };
     }
+    /**
+     * @swagger
+     * /users/current:
+     *    get:
+     *      description: Returns current authorized user
+     *      responses:
+     *        200:
+     *          description: Successful operation
+     *          content:
+     *            application/json:
+     *              schema:
+     *                type: object
+     *                items:
+     *                  $ref: '#/components/schemas/User'
+     */
+    private getCurrent({ user }: ApiHandlerOptions): ApiHandlerResponse {
+        return {
+            status: HttpCode.OK,
+            payload: user as UserGetCurrentResponseDto,
```

Comment:
It doesn't make sense, why do we accept user obj and then return the same user? We should find the user by id in the database and then return all properties (except for password hash and salt)

Reply:
Now i see, thanks